deployments/server/templates/ingress.yaml

```diff
@@ -0,0 +1,18 @@
+apiVersion: networking.k8s.io/v1
+kind: Ingress
+metadata:
+  name: {{ include "user-mananger-server.fullname" . }}-http
```

Comment:
Only defining an ingress for HTTP. It would be better if we can use gRPC from CLI, but I'm not sure if we can assume that ingress controllers can route gRPC requests. Kong can do, but I'm not sure for other ingress controllers.